compat/src/Children.js

```diff
@@ -1,8 +1,11 @@
 import { toChildArray } from 'preact';
 
-const mapFn = (children, fn) => {
-	if (children == null) return null;
-	return toChildArray(toChildArray(children).map(fn));
+const mapFn = (children, fn, context) => {
+	if (!children) return null;
+	return toChildArray(children).reduce(
```

Comment:
We don't generally use reduce(). I'd prefer to use `.map(fn.bind(ctx))` for size and perf reasons.